src/ghvmctl

```diff
@@ -19,49 +26,65 @@ prepare_vm() {
 		-d root,size="${VM_DISK_GIB}GiB"
 
 	# Wait for the VM agent to be running
-	while ! lxc exec test-vm -- cat /etc/hostname &>/dev/null; do
-		sleep 2
+	while ! lxc exec "${VM_NAME}" -- true 2>/dev/null; do
+		sleep 1
 	done
 
 	info "Installing utilities in VM"
 
+	# Try avoid gnome-initial-setup to start.
+	lxc exec "${VM_NAME}" -- runuser -u ubuntu -- \
+		sh -c 'echo yes > ~/.config/gnome-initial-setup-done'
+	lxc exec "${VM_NAME}" -- sh -c \
+		'rm /usr/lib/systemd/user/gnome-session.target.wants/gnome-initial-setup*'
+	lxc exec "${VM_NAME}" -- sh -c \
+		'rm /etc/xdg/autostart/gnome-initial-setup*'
+
 	# Push script runner into the VM
-	lxc -q file push "${SNAP}/opt/ghvmctl/ghvmctl-runner" test-vm/bin/ghvmctl-runner
-	lxc exec test-vm -- chmod 755 /bin/ghvmctl-runner
+	lxc -q file push "${GHVMCTL_RUNNER_PATH}/ghvmctl-runner" \
+		"${VM_NAME}/bin/ghvmctl-runner"
+	lxc exec "${VM_NAME}" -- chmod 755 /bin/ghvmctl-runner
 
-	lxc exec test-vm -- apt-get update -qq
-	lxc exec test-vm -- apt-get install -y gnome-screenshot
+	lxc exec "${VM_NAME}" -- sh -c \
+		"echo 'APT::Get::Assume-Yes \"true\";' > /etc/apt/apt.conf.d/90aptyes"
+	lxc exec "${VM_NAME}" -- apt-get update -qq
+	lxc exec "${VM_NAME}" -- apt-get install -q gnome-screenshot
 
-	# Kill the gnome initial setup wizard
-	pid="$(lxc exec test-vm -- pidof gnome-initial-setup)"
-	lxc exec test-vm -- kill -9 "$pid"
+	info "Configuring VM user settings"
+	exec_in_vm gsettings set org.gnome.desktop.lockdown disable-lock-screen true
+	exec_in_vm gsettings set org.gnome.desktop.session idle-delay 0
+
+	# Ensure initial-setup is not running, keep this last thing.
+	exec_in_vm systemctl --user stop "gnome-initial-setup*"
 }
 
 # Exec a command in the VM using the wrapper.
 exec_in_vm() {
-	lxc exec test-vm -- sudo -u ubuntu bash -c "/bin/ghvmctl-runner $*"
+	lxc exec "${VM_NAME}" -- sudo -u ubuntu bash -c "/bin/ghvmctl-runner $*"
 }
 
 # Takes a screenshot of the full screen and pulls the file back from the VM to a file
 # named "screenshot-screen.png", and uploads to imgur, returning the URL
 screenshot_full() {
 	info "Taking a screenshot of the full screen inside the VM"
-	mkdir -p "${SNAP_REAL_HOME}/ghvmctl-screenshots"
+	mkdir -p "${REAL_HOME}/ghvmctl-screenshots"
 	# Take a screenshot of the whole screen in the VM
 	exec_in_vm "gnome-screenshot -f /home/ubuntu/screen.png"
 	# Pull the screenshot back to the host
-	lxc file pull test-vm/home/ubuntu/screen.png "${SNAP_REAL_HOME}/ghvmctl-screenshots/screenshot-screen.png"
+	lxc file pull "${VM_NAME}/home/ubuntu/screen.png" \
+		"${REAL_HOME}/ghvmctl-screenshots/screenshot-screen.png"
 }
 
 # Takes a screenshot of the active window and pulls the file back from the VM to a file
 # named "screenshot-window.png", and uploads to imgur, returning the URL
 screenshot_window() {
 	info "Taking a screenshot of the active window inside the VM"
-	mkdir -p "${SNAP_REAL_HOME}/ghvmctl-screenshots"
+	mkdir -p "${REAL_HOME}/ghvmctl-screenshots"
```

Comment:
Same here

Reply:
Depends if we want to still support running this outside the snap for development, we can replace the location of the place not having a new variable for it.